src/groups/mqb/mqbs/mqbs_filestore.h

```diff
@@ -876,6 +882,12 @@ class FileStore : public DataStore {
     /// builder is empty (just flushed).
     void notifyQueuesOnReplicatedBatch();
 
+    /// When reconfigure from weak consistency to strong consistency:
+    /// Cancel replication notification (if any) for the specified `queueKey`.
+    /// No effect if `queueKey` is not found in this storage or if there are no
+    /// current registered notifications for this queue.
+    void cancelReplicationNotification(const mqbu::StorageKey& queueKey);
```

Comment:
This early return is helpful because it prevents us from doing this lookup a few lines below:

```
    RecordHandleMap::const_iterator it = d_handles.find(msgGUID);
```

If we can do early exit without accessing `OrderedHashMapWithHistory`, it will be nice.